synth.py

```diff
@@ -30,7 +30,7 @@
   name, 
   version,
   bazel_target=f"//google/appengine/{version}:appengine-{version}-nodejs")
-  s.copy(library, excludes=["package.json","linkinator.config.json", "system-test/fixtures/sample/src/index.js","system-test/fixtures/sample/src/index.ts"])
+  s.copy(library, excludes=["package.json","README.md"])
```

Comment:
Do we still need the custom README and explicit package.json opt out? These feel weird.

Reply:
Hm, I can see how maybe we don't need to opt out the README, but wouldn't we still want the package.json opted out (and use renovate bot instead)?